src/applications/accreditation/21a/tests/fixtures/data/test-data.json

```diff
@@ -85,7 +85,7 @@
         "reasonForLeaving": "Found a better opportunity"
       }
     ],
-    "activities": ["financial", "home"],
+    "employmentActivities": ["FINANCIAL", "HOME"],
```

Comment:
this is the only thing that I think will break, as the keys in the options are still lower case. happy to merge this PR as-is and update in the other one